ambry-commons/src/main/java/com.github.ambry.commons/HelixNotifier.java

```diff
@@ -42,23 +46,21 @@
 public class HelixNotifier implements Notifier<String> {
   private static final String TOPIC_PATH = "/topics";
   private static final String MESSAGE_KEY = "message";
+  private static final Logger classLogger = LoggerFactory.getLogger(HelixNotifier.class);
```

Comment:
Would follow a naming convention that `ALL_CAPS` are used for `static final` primitives, and `camelCase` for non-primitives. [Link](https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case)